rest/config.go

```diff
@@ -331,7 +334,122 @@ func (dbConfig *DbConfig) setup(name string) error {
 		dbConfig.Server = &urlStr
 	}
 
-	return err
+	// Load Sync Function.
+	if dbConfig.Sync != nil {
+		sync, err := loadJavaScript(*dbConfig.Sync)
+		if err != nil {
+			return JavaScriptLoadError{
+				Message: "error loading sync function",
```

Comment:
Instead of message, define 'type' as an enum for sync function/conflict resolver/import filter.

Reply:
Introduced a an enum for sync function/conflict resolver/import filter and pushed the latest revision.